Shine/Shine.swift

```diff
@@ -206,7 +206,7 @@ fileprivate extension Shine {
 			return
 		}
 		let displayVersion = infoDictionary["CFBundleShortVersionString"] as? String
-		let bundleDisplayname = infoDictionary["CFBundleName"] as? String
+		let bundleDisplayname = Bundle.main.infoDictionary?["CFBundleDisplayName"] as? String
```

Comment:
Let's modify this to reference the infoDictionary var instead of accessing directly from the main bundle. This is just to match the other accessed keys.